app/manage-your-appointments/2025/10/pharmacy-visits-what-we-learnt-part-1/index.md

```diff
@@ -0,0 +1,54 @@
+
+These approaches worked okay, but still felt like ‘workarounds’ with trade-offs in either appointment availability (approaches 1 and 2) or maintenance overheads (approach 3). One approach some of these pharmacies asked about was setting a maximum number of vaccination appointments that could be booked each day. That way, the appointment slots could match their real opening hours and the actual time it took to do the vaccination, without the risk of the entire day getting booked up with vaccinations leaving no time for anything else. 
+
+## Assumptions challenged
+In addition to helping us understand how pharmacies run their services, this research challenged a few unspoken assumptions I think we’d been making.  
+
+### 1. Professionals are more confident with technology 
+We assumed that because our users are professionals, with higher level qualifications, they’ll be “tech savvy” and comfortable with more technical language than we might include in a public facing service. In reality, that wasn’t the case. In general, as long as the task was straightforward they were okay, but when things got a little more tricky it caused problems. For example, many pharmacies really struggled to track down the service welcome email using their inbox search. 
+
+### 2. They'll read guidance or attend training first 
+We’d assumed staff would attend training or read guidance before using the service. We also thought they’d approach creating availability carefully and methodically, with a mindset more like doing a tax return, than doing online shopping. 
+
+In reality pharmacy staff simply don’t have the time for guidance or training beforehand, or to carefully double check everything. Plus, finding time without distractions to do an admin task like this is hard. That means when they get to it, the aim is simply to get their appointment availability set up as quickly as possible, and before they get interrupted by something else. 
+
+### 3. Pharmacies run services to the minute 
+We assumed pharmacies deliver most vaccinations at pre-booked time slots, sticking closely to scheduled appointments. At the pharmacies we visited in this research, we found walk-ins were equally, if not more common. My impression was that pharmacies are less concerned with strict timings and more focused on providing a good service overall. If someone turns up early, they may be seen early; if they’re late, it’s usually not a problem. 
+
+When you take a step back, none of this is especially surprising. But, without regular research with end users, it’s easy to slip back into self-referential design and start making these kinds of assumptions. 
+
+## Opportunities for improvement
+During these visits we also observed pharmacy staff using MYA for the first time. This highlighted several usability issues, which will be outlined in another design history entry coming soon.
```

Comment:
You could drop this paragraph and just mention it on the next post instead? (Much as I like the idea of teasing the next post!)

```suggestion
```